recipe/meta.yaml

```diff
@@ -35,8 +35,14 @@ test:
 about:
   home: https://github.com/null-jones/streamlit-plotly-events
   summary: Plotly chart component for Streamlit that also allows for events to bubble back up to Streamlit.
+  description: |
+    Plotly chart component for Streamlit that also allows for events to bubble back up to Streamlit.
+    Ever wanted to harness those awesome events from Plotly charts inside of Streamlit? So did I, so now you can!
```

Comment:
```suggestion
```
This doesn't add any useful information and is more marketing content. (I know it comes from upstream, but we don't need to add this here).

Reply:
sorry, sometimes copy paste takes it too far.